From 9455b358ccecf5b9e493a21eea6748e844478db5 Mon Sep 17 00:00:00 2001
From: Steve Loughran <stevel@apache.org>
Date: Thu, 1 Dec 2016 11:26:59 +0000
Subject: [PATCH 2219/2848] HADOOP-13850 s3guard to log choice of metadata
 store at debug. Contributed by Mingliang Liu

(cherry picked from commit cfd0fbf13b7e901d991456f14ff7d4b89301f388)

Change-Id: I5d4564db056e3c82391a180431114072cde1bc11
---
 .../org/apache/hadoop/fs/s3a/s3guard/S3Guard.java  |    5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/s3guard/S3Guard.java b/hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/s3guard/S3Guard.java
index c998072..904a1c3 100644
--- a/hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/s3guard/S3Guard.java
+++ b/hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/s3guard/S3Guard.java
@@ -66,7 +66,6 @@ private S3Guard() { }
    * @return Reference to new MetadataStore.
    */
   public static MetadataStore getMetadataStore(FileSystem fs) {
-
     Preconditions.checkNotNull(fs);
     Configuration conf = fs.getConf();
     Preconditions.checkNotNull(conf);
@@ -74,8 +73,8 @@ public static MetadataStore getMetadataStore(FileSystem fs) {
     try {
       Class<? extends MetadataStore> msClass = getMetadataStoreClass(conf);
       msInstance = ReflectionUtils.newInstance(msClass, conf);
-      LOG.info("Using {} for {} filesystem", msClass.getSimpleName(),
-          fs.getScheme());
+      LOG.debug("Using {} metadata store for {} filesystem",
+          msClass.getSimpleName(), fs.getScheme());
     } catch (RuntimeException e) {
       LOG.error("Failed to instantiate {}, using NullMetadataStore:",
           conf.get(S3_METADATA_STORE_IMPL), e);
-- 
1.7.9.5

